pkg/secrets/repositoryserver/aws_location.go

```diff
@@ -0,0 +1,34 @@
+package repositoryserver
+
+import (
+	"github.com/pkg/errors"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	AWSBucketKey string = "bucket"
+	AWSRegionKey string = "region"
```

Comment:
These are also defined as constants here:
https://github.com/kanisterio/kanister/blob/f74570f64b56527abd7e6440a1b8a328465497ce/pkg/kopia/command/storage/secret_utils.go#L34-L39

We can use common constants for these, this might require some refactoring.

Reply:
yes this is going to need refactoring as these are not exported. I will raise another PR to reuse them. If thats okay?

Reply:
Done!